src/booking/BookingsModal.tsx

```diff
@@ -166,6 +172,7 @@ const BookingsModal: FunctionComponent<BookingsModalProps> = ({
                             },
                           },
                         }}
+                        onClick={() => enqueueAnalyticsEvent('Click make a booking')}
```

Comment:
Can we include maybe information about the booking? i.e. the date, far in the future the booking is, what location etc.

Reply:
done